modules/post/android/gather/wireless_ap.rb

```diff
@@ -0,0 +1,87 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  Rank = NormalRanking
```

Comment:
`Rank` doesn't really make sense for `Post` modules.

Only five `Post` modules have a `Rank`. Of those, two relate to privilege escalation/persistence where a `Rank` kind of makes sense. The remaining three likely only have a `Rank` due to accidental copy/paste.